lib/bot.js

```diff
@@ -148,6 +168,13 @@ class Bot {
     return this.commandCharacters.indexOf(message[0]) !== -1;
   }
 
+  static substitutePattern(message, patternMapping) {
+    return message.replace(patternMatch, (match, varName) => {
+      if (varName in patternMapping) return patternMapping[varName];
```

Comment:
The main reason I didn't was in case the patternMapping was a blank string. I guess this probably isn't going to happen anywhere (unless there was somehow a zero-length discord message? But there's a guard for that), so it should be fine to simplify.